lib/iiif_print/split_pdfs/child_work_creation_from_pdf_service.rb

```diff
@@ -35,7 +35,7 @@ def self.conditionally_enqueue(file_set:, file:, user:, import_url: nil, work: n
         return :no_pdfs if file_locations.empty?
 
         work.iiif_print_config.pdf_splitter_job.perform_later(
-          work,
+          file_set,
```

Comment:
Hmm, i'm not understanding why this is changing from the work to fileset here

Reply:
In the job (that this is calling) we have seen errors where the `work` is `nil`.  Why is it nil?  Because we never check the results of `parent_for`.  Further confounding this problem, is that a job with a `nil` work will be infinitely rescheduled and never recoverable.

By shifting to the file_set, we create a more recoverable situation (or at least discernable if the parent never "arrives" in the data).

In future jobs, I'm assuming we're going to always receive the file_set (because we use that to help us with the Derivative Rodeo).

Another way to think about it:  we can derive a file_set's parent work easier than we can derive the right file_set from a given work (which might be nil).  Thus we want to pass the more generally useful parameter.